dir.targets

```diff
@@ -61,5 +61,6 @@
   <PropertyGroup Condition="$(AssemblyOriginatorKeyFile.Contains('Open.snk'))">
     <StrongNameSig />
     <DelaySign>false</DelaySign>
-  </PropertyGroup> 
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
```

Comment:
I think you're saying set DelaySign to true and also set SignType to "oss" so sign.targets in buildtools ends up doing the open source singing?